File: math/fibonacci-series/fibonacci-series.py
value = int(input("Enter a value..."))
x1 = -1
x2 = 1
x3 = 0
print("The fibonacci series is...")
for x in range(value):
	x3 = x1 + x2
	x1 = x2
	x2 = x3
	print(x3)

File: math/cpp/fibonacci-series/fibonacci-series.c
#include<stdio.h>
void main()
{
	int i, n , x1 = -1, x2 = 1, x3 = 0;
	printf("Enter a valuee...\n");
	scanf("%d",&n);
	printf("The fibonacci series till %d are\n",n );
	for(i = 0; i < n ; i++)
	{
		x3 = x1 + x2;
		x1 = x2;
		x2 = x3;
		printf("%d\n",x3);
	}

}